lib/ueberauth/strategy/google/google.ex

```diff
@@ -45,7 +45,8 @@ defmodule Ueberauth.Strategy.Google do
         |> put_non_nil(:access_type, Map.get(params, :access_type))
 
       url =
-        (opts ++ [redirect_uri: url])
+        opts
```

Comment:
Adding a single element to the tail of a list with `++` is non ideal.  We should either add it to the head or we can use something like `Keyword.put/3`.